src/components/Resources/ResourceCard.js

```diff
@@ -118,3 +119,11 @@ export const ResourceCard = ({ id, title, created, description, url }) => {
     </Card>
   );
 };
+
+ResourceCard.propTypes = {
+  id: PropTypes.number,
```

Comment:
Thanks @peoray! One note here: I think `id` should be a string, not a number, according to https://github.com/codebuddies/frontend/blob/master/db.json.

Reply:
That's on me -- I told him that it should be IDs. I didn't realize that by IDs, we mean GUIDs? https://github.com/codebuddies/backend/blob/6e43e73f9a22c7a8a10d03d3ccae42f679b643de/cbv3_django_prototype/cbv3_django_prototype/resources/models.py#L34

In any case, I think this is fine for now, and we can fix it up once we're able to start fetching resources from the API.

Reply:
No worries, it's a quick/small change that can be folded elsewhere for sure :) 

Can't wait until we have api.codebuddies.org/v1 set up!